cli/stubs/ifconfig.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>Label</key>
+    <string>com.laravel.valet.ifconfig</string>
```

Comment:
Heretofore, we've been able to say that "Valet doesn't run as a service". 
I worry that now this adds a plist that people will conflate with being a service, and will think that valet is depending on this, or that it is broken if this doesn't exist.

Definitely going to add to the complexities of trying to support this whole feature.

Reply:
This could be removed with adding documentation about persistent loopback interface alias.